tests/cli/set/test_sample.py

```diff
@@ -2,13 +2,11 @@
 import logging
 
 import pytest
-from click.testing import CliRunner
-
 from cg.cli.set.base import sample
+from cg.constants import Priority
 from cg.models.cg_config import CGConfig
 from cg.store import Store
-
-from cg.constants import Priority
+from click.testing import CliRunner
 
 SUCCESS = 0
```

Comment:
Same here. Let's clean it up by importing the constant instead of redefining it again

Reply:
Are you referring to the constant being assigned to the "new_value" variable unnecessarily? In that case, is there any argument for it the current state being more readable, as key and new_value? 

![image](https://user-images.githubusercontent.com/14308912/203982849-2e2113cc-8f47-4445-9fef-e8221aa917e8.png)

Reply:
I mean like this:
```
from cg.constants import EXIT_SUCCESS
```
And the just use that variable. Then you keep it DRY (do not repeat yourself)

Reply:
I see, then I had made the changes already from the earlier comment : ) Resolving!